README.md

```diff
@@ -15,9 +15,16 @@ With Ruby installed on your machine, you can install [Bundler] using the command
 
 Next, use [Bundler] to install [Jekyll] and other dependencies needed to run the website.
 
-    git clone https://github.com/apache/accumulo-website
-    cd accumulo-website
-    bundle install
+     git clone https://github.com/apache/accumulo-website
+     cd accumulo-website
+     
+     # On Ubuntu (PopOS) perform the next three commands
```

Comment:
Instead of adding this to this section, it'd be better to add this to the paragraph above that talks about OS-packaged versions of ruby already. After the sentence mentioning ruby-dev and ruby-devel, you could add the sentence:

> Depending on your OS, you may also need other packages, such as `ruby-full`, `make`, `gcc`, `nodejs`, `build-essentials`, or `patch`.